docs/guides/user-guide/available-environments.md

```diff
@@ -15,13 +15,15 @@ Depending on the environment, StackBlitz IDE includes slightly different feature
 
 | Feature | EngineBlock | WebContainers |
 | --- | --- | --- |
-| Native Node.js environment | <SupportIcon value="no" label="Not available" /> | <SupportIcon value="yes" label="Available" /> |
 | Supported frameworks | <SupportIcon value="star-half" label="" /> Front-end | <SupportIcon value="star" label="" /> Front-end & back-end |
-| Editor | <SupportIcon value="yes" label="Available" /> | <SupportIcon value="yes" label="Available" /> |
-| Preview | <SupportIcon value="yes" label="Available" /> | <SupportIcon value="yes" label="Available" /> |
-| Shareable Preview URL | <YepNope value="yes" label="Supported" /> | <SupportIcon value="no" label="Not supported" /> |
-| Console | <SupportIcon value="yes" label="Available" /> | <SupportIcon value="no" label="Not available" /> |
-| Terminal | <SupportIcon value="no" label="Not available" /> | <SupportIcon value="yes" label="Available" /> |
+| Supported package managers | <SupportIcon value="star-half" label="" /> Turbo v1 | <SupportIcon value="star" label="" /> Turbo v2, pnpm, yarn v1 |
```

Comment:
I'm not sure how accurate the package manager information is at this time.

Also mentioning the Turbo package manager might be a bit arcane for readers, and we don't have great pages to link to. There is [this article on Medium about Turbo v1](https://medium.com/stackblitz-blog/introducing-turbo-5x-faster-than-yarn-npm-and-runs-natively-in-browser-cc2c39715403), and [this page in our docs about Turbo v2 but it's a bit messy](https://developer.stackblitz.com/platform/webcontainers/turbo-package-manager).

Reply:
Folks really often ask about package managers so I think we should leave it but probably improve the docs page? Wdyt?